src/Channel.cpp

```diff
@@ -92,7 +96,8 @@ JointState Channel::getJointState() const {
         case CONTROL_PROFILED_POSITION:
         case CONTROL_POSITION: {
             int32_t feedback = get<Feedback>();
```

Comment:
get<Feedback> here is wrong if the source is the encoder ... the previous code was right.